packages/stores/src/ui-config/trade-token-in-config.ts

```diff
@@ -410,6 +437,9 @@ export class ObservableTradeTokenInConfig extends AmountConfig {
     }
 
     if (this.amount) {
+      if (this._error instanceof NoRouteError) return this._error;
+      if (this._notEnoughLiquidity) return new NotEnoughLiquidityError();
+
```

Comment:
I think this needs to be moved below the balance code below (below that last if statement).

Reply:
If I moved it down there, `Insufficient Balance` would take precedence. Users who like to experiment can get confused when the `out` token stops providing estimates, just like it happened to Jim. I think leaving it like it's right now will benefit both users. What do you think?

Reply:
I guess that's true given that we don't calculate past the pool's liquidity as other DEXs do. However, it's not ideal to be advertising that we don't have enough liquidity. Both are true to some extent.

SGTM